src/main/webapp/help-gradleInjectionDisabledNodes.html

```diff
@@ -0,0 +1,4 @@
+<div>
+    The list of node labels where the Gradle Enterprise Gradle Plugin injection should be disabled.
+    By default, all nodes are enabled.
```

Comment:
maybe add "Disabled nodes have precedence over enabled nodes", in this help, and the 'enabled help' below

Reply:
Done.